.gitignore

```diff
@@ -0,0 +1,7 @@
+/node_modules
+*.sw[po]
+/store
+/.cargo
+/dist
+.env
+keypair.json
```

Comment:
.DS_Store is not relevant to the project, probably should delete it from the repo and add it to `.gitignore`

Reply:
ok

Reply:
ok